kernel/build-kernel.sh

```diff
@@ -138,9 +146,9 @@ get_default_kernel_config() {
 	[ -n "${hypervisor}" ] || die "hypervisor not provided"
 	[ -n "${kernel_arch}" ] || die "kernel arch not provided"
 
-	major_version=$(echo "${version}" | cut -d. -f1)
-	minor_version=$(echo "${version}" | cut -d. -f2)
-	config="${default_kernel_config_dir}/${kernel_arch}_kata_${hypervisor}_${major_version}.${minor_version}.x"
+	local major_kernel
+	major_kernel=$(get_major_kernel_version "${version}")
```

Comment:
Not a functional thing, but it would be nice to change the variable name to ${kernel_ver} since we do have both major and minor version in that variable but it is called 'major_kernel'

Reply:
Thanks, changed.